.changeset/rich-emus-say.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": patch
+---
+
+Temporarily disable streaming for improved model compatibility - will be re-enabled in upcoming release
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)